go/base/static/js/test/local.test.js

```diff
@@ -0,0 +1,36 @@
+describe("go.local", function() {
+  afterEach(function() {
+    localStorage.clear();
+  });
+
+  describe(".set", function() {
+    it("should stringify and set values", function() {
+      assert(!('foo' in localStorage));
+      go.local.set('foo', {bar: 23});
+      assert.deepEqual(JSON.parse(localStorage.foo), {bar: 23});
+    });
+  });
+
+  describe(".get", function() {
+    it("should return null for non-existent items", function() {
+      assert(!('foo' in localStorage));
+      localStorage.bar = JSON.stringify({baz: 21});
+
+      assert.strictEqual(go.local.get('foo'), null);
+      assert.notStrictEqual(go.local.get('bar'), null);
+    });
+
+    it("should support default values", function() {
+      assert(!('foo' in localStorage));
+      localStorage.bar = JSON.stringify({baz: 21});
+
+      assert.strictEqual(go.local.get('foo', 23), 23);
```

Comment:
We are asking `.get()` to give us the current local storage value for the key `'foo'`, but asking it to instead return `23` if `'foo'` does not exist. Does that help?